packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart

```diff
@@ -187,19 +187,19 @@ class BuildDaemonCreator {
     bool initializePlatform,
     WebTestTargetManifest testTargets,
   }) {
+    // The build script is stored in an auxillary package to reduce
+    // dependencies of the main tool.
     final String flutterToolsPackages = globals.fs.path.join(
       Cache.flutterRoot,
       'packages',
-      'flutter_tools',
+      '_flutter_web_build_script',
       '.packages',
```

Comment:
As discussed offline, test for the existence of this file and fail in a nice way if it's not there.